### ISSUE TITLE
Parser for lambda lists

### ISSUE THREAD

**Issue by phoe** [commander-trashdin/clng#5]
> There are a lot of generic half-baked ones. It also may include the discussion of what can be added/changed about lambda-lists structure – such as pattern matching.

https://github.com/s-expressionists/Ecclesia includes such utilities as a library.

**Comment by commander-trashdin** [comment]
Right, I (or you) can add that to the Current status part. But the "lambda-lists structure" is more of a point, I guess I maybe shouldn't dismiss the existing libraries as half-baked, or at least do it less. I would want a point like that, maybe more general, referring to how variables are declared/bound at all....I struggle to word that precisely.
